### DISCUSSION TITLE
科技爱好者周刊（第 254 期）：人生是一个长板问题 | ruanyf-weekly

### DISCUSSION THREAD

**Discussion by giscus** [plantree/press-comment#16] [General]
# 科技爱好者周刊（第 254 期）：人生是一个长板问题 | ruanyf-weekly

阮一峰的技术周刊

https://ruanyf-weekly.vercel.app/weekly/issue-254

**Comment by terminalqo** [comment]
提个建议，弄个微信二维码分享到朋友圈。这样能导流。
